lib/dor/workflow_archiver.rb

```diff
@@ -25,8 +25,9 @@ def to_bind_hash
       h
     end
 
-    def set_current_version
-      self.version = Faraday.get WorkflowArchiver.config.dor_service_uri + "/dor/v1/objects/#{druid}/versions/current"
+    def set_current_version(dor_conn)
+      response = dor_conn.get "/dor/v1/objects/#{druid}/versions/current"
```

Comment:
should this be some sort of config/const value?  Even if it's a yml file without using config gem?

I'm not sure ... just want to raise the question.   I see the var in the middle, which makes it seemingly easier as an interpolated string (like we use in pohandler)

Reply:
The closest I could get to your suggestion was like this:
```ruby
def current_druid_version_url
  "/dor/v1/objects/#{self.druid}/versions/current"
end
# and then later...
response = dor_conn.get current_druid_version_url
```
If i tried a class constant like so:
```ruby
CURRENT_DRUID_VERSION_URL = "/dor/v1/objects/#{self.druid}/versions/current"
# and then later...
response = dor_conn.get CURRENT_DRUID_VERSION_URL
```
I get a
```
NoMethodError:
  undefined method `druid' for #<Class:0x00007fc8b1814c10>
```
Not sure I follow your suggestion to use a yml file, but what do you think about the `current_druid_version_url` method approach?

Reply:
I was thinking more like

at top of class (?)

```
DOR_CURRENT_VERSION_PATH = "/dor/v1/objects/ %{druid}/versions/current"
```

and then in situ:
```
response = dor_conn.get(DOR_CURRENT_VERSION_PATH, druid)
```

similar to what is done in PreservedObjectHandler in PC code, or similarly in Validator in MV.